### DISCUSSION TITLE
multiple experiments are not running in the file

### DISCUSSION THREAD

**Discussion by sanket-mishra-profile** [quaquel/EMAworkbench#329] [Q&A]
in my experimentation, i  want to try multiple scenario as well as multiple policies , which leads to multiple experiments
![image](https://github.com/quaquel/EMAworkbench/assets/86866430/903bdf98-cc94-455d-8f8b-0ff8e655a74b)

but I dont have any idea, why multiple experiments are not running 
could you let me know what could be the reason
I have to use EMA workbench for my research.

Also, are there any other workbenchs/library which can be used for EMA instead of this one (where the model is based on VENSIM software only for which EMA has to be performed)? Please let me know

**Comment by quaquel** [comment]
You now have asked three questions within 15 minutes. I am happy to help, but there are some basic ground rules. 

Please check https://stackoverflow.com/help/how-to-ask. Next, work your way through some simple, pure python examples. And only then move on to what you are trying to do.

**Comment by EwoutH** [comment]
A [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) would be very useful to help you further with this issue.

Since this is likely not an issue with the workbench, I'm going to convert this to a Discussion.
